### ISSUE TITLE
Automatic update deleted everything

### ISSUE THREAD

**Issue by konradjurk** [getinsomnia/insomnia#6917]
### Expected Behavior

Open Insomnia, find usual workspace

### Actual Behavior

Opened Insomnia, which updated automatically (I did not configure that), found confusing screens, with confusing text, and confusing options to log in and store things somewhere or local, chose local, and found an empty workspace. Upon importing the base collections again and inserting login tokens, Insomnia presents an error message which is not very helpful.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

<img width="391" alt="Bildschirmfoto 2023-12-11 um 18 20 07" src="https://github.com/Kong/insomnia/assets/6976170/25dd300e-6873-455e-aca7-6872ce52ac4c">

### Insomnia Version

Version: Insomnia 8.4.5 Build date: 11/23/2023 OS: Darwin arm64 22.6.0 Electron: 27.0.3 Node: 18.17.1 Node ABI: 118 V8: 11.8.172.16-electron.0 Architecture: arm64

### What operating system are you using?

macOS

### Operating System Version

MacOS Ventura 13.6.1 (22G313)

### Installation method

downloaded

**Comment by Uepsilon** [comment]
Same result here, found a workaround to at least restore everything
1. go to the account menu
<img width="379" alt="image" src="https://github.com/Kong/insomnia/assets/452717/dd34867c-872d-429e-98e3-309e99e17130">

2. click export data and more
<img width="617" alt="image" src="https://github.com/Kong/insomnia/assets/452717/57958ed2-e14d-4b0c-ac9e-e2d8216fca14">

3. export everything
<img width="565" alt="image" src="https://github.com/Kong/insomnia/assets/452717/adc96eb3-95a8-4a08-821c-d1b1737192b3">

4. go back to scratchpad overview, click on the arrow down next to it and choose import
<img width="252" alt="image" src="https://github.com/Kong/insomnia/assets/452717/0642055b-8b46-4f24-bbf9-92603c0b9155">

5. import the previously exported file, that does has scratchpad in the name. this restored my data.

kinda weird behaviour to be honest. 

also, that only restores the data. Making requests don't work with the same error message as above. 

_I've added this more as a guide on how to restore the data_

**Comment by Uepsilon** [comment]
okay, I figures, that in my case there was a variable defined, that was not in the Base Environment but if I select an Sub Environment it works again. 

missing variables are highlightes by a red ! next to the name, so look out for these
<img width="35" alt="image" src="https://github.com/Kong/insomnia/assets/452717/29b5d368-e7e4-4197-bac6-697c2b21f9c7">

**Comment by subnetmarco** [comment]
Yes this is a very hard to understand message that points to variables that are not valued, could be more user friendly.
